### ISSUE TITLE
Can't start webview if no blockchain is present

### ISSUE THREAD

**Issue by r4sas** [Revertron/Alfis#10]
On Windows with default webwiew feature (not edge):

```
2021-04-01 12:42:42,752 INFO  [alfis::Main] Starting ALFIS 0.3.7
2021-04-01 12:42:42,753 INFO  [alfis::Main] Loaded settings: Settings { origin: "00000102C2F9BFD2803284D93327F089D60FC72A06F19AF2384567F2646B8348", key_file: "default.key", net: Net { peers: ["test-ip4.alfis.name:4244", "test-ip6.alfis.name:4244"], listen: "[::]:4244", public: true, yggdrasil_only: false }, dns: Dns { listen: "127.0.0.1:5300", threads: 20, forwarders: ["127.0.0.1:53"] }, mining: Mining { threads: 4 } }
2021-04-01 12:42:42,794 INFO  [alfis::Main] No blocks found in DB
2021-04-01 12:42:42,799 DEBUG [alfis::dns::authority] Authority dir (zones) not found, skipping.
2021-04-01 12:42:42,804 DEBUG [alfis::p2p::network] Started node listener on [::]:4244
2021-04-01 12:42:42,906 DEBUG [alfis::web_ui] Command {"cmd":"loaded"}
2021-04-01 12:42:42,948 INFO  [alfis::web_ui] Current blockchain height is 0
```
![image](https://user-images.githubusercontent.com/7023612/113275886-ed397b00-92e7-11eb-9c3f-904f9196af23.png)
After answering Yes or No you will get error and application will be stopped:
```
2021-04-01 12:42:43,716 ERROR [alfis::web_ui] Something wrong with webview, exiting
```

**Comment by r4sas** [comment]
Can be same: https://github.com/Revertron/Alfis/issues/7#issuecomment-810339548

**Comment by acetoneRu** [comment]
I have this bug too.

**Comment by Revertron** [comment]
Thanks for the report!
Fixed in commit 11bc887fbb5051f6ad30c5d5f234185cf53ea914.
